components/Cases/CaseRecap/CaseRecap.tsx

```diff
@@ -1,35 +1,21 @@
-import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
-import Spinner from 'components/Spinner/Spinner';
-import Summary from 'components/Summary/Summary';
-import { useCase } from 'utils/api/cases';
-import * as form from 'data/forms';
-import { FormStep } from 'components/Form/types';
+import CaseNote from 'components/Cases/CaseRecap/CaseNote';
+import HistoricNote from 'components/Cases/CaseRecap/HistoricNote';
 
 interface Props {
   personId: number;
   recordId: string;
+  is_historical: true | false;
 }
 
-const CaseRecap = ({ personId, recordId }: Props): React.ReactElement => {
-  const { data: record, error: recordError } = useCase(recordId);
-  const recordData = record?.caseFormData?.form_name_overall;
-  const fileData =
-    recordData && (form as Record<string, FormStep[]>)[recordData];
-  if (recordError || (recordData && !fileData)) {
-    return <ErrorMessage />;
+const CaseRecap = ({
+  personId,
+  recordId,
+  is_historical,
+}: Props): React.ReactElement => {
+  if (is_historical) {
```

Comment:
we can do this in the `page` component, so we don't have a component just for an `if`